tasks/human_eval_024.rs

```diff
@@ -5,11 +5,98 @@ HumanEval/24
 /*
 ### VERUS BEGIN
 */
+use vstd::arithmetic::div_mod::{
+    lemma_fundamental_div_mod, lemma_fundamental_div_mod_converse_div,
+};
 use vstd::prelude::*;
 
 verus! {
 
-// TODO: Put your solution (the specification, implementation, and proof) to the task here
+pub open spec fn mul(a: nat, b: nat) -> nat {
+    builtin::mul(a, b)
+}
+
+/// Specification for what it means for d to divide a
+pub open spec fn divides(v: nat, d: nat) -> bool {
```

Comment:
Can you reverse the ordering of the arguments in `divides`, so that it matches the English connotation, i.e., so `divides(d, v)` matches "d divides v"?

Reply:
For some reason, one example in verus also follows the less intuitive ordering https://github.com/verus-lang/verus/blob/bbc3e9d68b64f715b62ba46378e859cf0534d1fc/source/rust_verify/example/assorted_demo.rs#L31-L33